### PR TITLE
spelling fixes (round 5)

### PR DESCRIPTION
**Pull request by jsoref**
(For a list of the fixes, see the commit message)

### CODE REVIEW

File: src/harness/compilerRunner.ts [hunk 1]
```diff
@@ -262,7 +262,7 @@ class CompilerBaselineRunner extends RunnerBase {
                     // different order with 'pull' operations, and thus can produce slightly differing
                     // output.
                     //
-                    // For example, with a full type check, we may see a type outputed as: number | string
+                    // For example, with a full type check, we may see a type displayed as: number | string
                     // But with a pull type check, we may see it as:                       string | number
```
Comment (jsoref) [inline src/harness/compilerRunner.ts]: I should reindent the tail of this line

File: src/services/utilities.ts [hunk 2]
```diff
@@ -265,7 +265,7 @@ namespace ts {
     }
 
     /* Gets the token whose text has range [start, end) and position >= start
-     * and (position < end or (position === end && token is keyword or identifier or numeric\string litera))
+     * and (position < end or (position === end && token is keyword or identifier or numeric\string literal))
```
Comment (jsoref) [inline src/services/utilities.ts]: Is there a reason this is using `\` instead of `/`?

Comment (RyanCavanaugh) [inline src/services/utilities.ts]: Should be `/`

File: src/services/formatting/formattingScanner.ts [hunk 3]
```diff
@@ -181,7 +181,7 @@ namespace ts.formatting {
                 // readTokenInfo was called before with the same expected scan action.
                 // No need to re-scan text, return existing 'lastTokenInfo'
                 // it is ok to call fixTokenKind here since it does not affect
-                // what portion of text is consumed. In opposize rescanning can change it,
+                // what portion of text is consumed. In opposize (sic) rescanning can change it,
```
Comment (RyanCavanaugh) [inline src/services/formatting/formattingScanner.ts]: @vladima ?

**Comment by RyanCavanaugh** [comment]
:+1:

File: src/services/formatting/formattingScanner.ts (hunk 3 above)
Comment (sandersn) [inline src/services/formatting/formattingScanner.ts]: I suspect it's supposed to be "In contrast".

Comment (vladima) [inline src/services/formatting/formattingScanner.ts]: `in opposite` (`in contrast` should work as well)?

Comment (DanielRosenwasser) [inline src/services/formatting/formattingScanner.ts]: Vlad's confirmed to go with "In contrast". @jsoref can you fix that up?

**Comment by jsoref** [comment]
@RyanCavanaugh: sorry for the delay

**Comment by DanielRosenwasser** [comment]
Beautiful :+1:

**Pull request merged**